src/wallet.ts

```diff
@@ -160,6 +166,25 @@ export function createWalletMiddleware({
     res.result = await processEthSignMessage(msgParams, req);
   }
 
+  async function ethSignTransaction(
```

Comment:
Could you move this function back up to the "transaction signatures" section?

Reply:
I have moved this function next to sendTransaction/processTransaction and added the testcases.